src/index.ts

```diff
@@ -98,7 +129,7 @@ export class IDBDatabaseConnector implements IKeyValueDatabaseConnector {
     }
 
     if (!this.hasTable(tableName)) {
-      throw new Error(`Table ${tableName} does not exist`);
+      throw new Error(`Table '${tableName}' does not exist`);
```

Comment:
Actually Vince pointed out that we cannot create tables in IndexedDB so this should always throw an `operation unsupported` error